### ISSUE TITLE
Button active state not showing (via Keyboard)

### ISSUE THREAD

**Issue by Khaan25** [unkeyed/unkey#2506]
### Preliminary Checks

- [X] I have reviewed the documentation: https://unkey.com/docs

- [X] I have searched for existing issues: https://github.com/unkeyed/unkey/issues

- [X] This issue is not a question, general help request, or anything other than a bug report directly related to Unkey. Please ask questions in our Discord community: https://unkey.com/discord.

### Issue Summary

When I navigate on the page using `Tab` key, Buttons get focused but I dont' see it visually.
There's search input which get's border when active, focused.

I was expecting some border around image that it's active, focused but I don't.

### Steps to Reproduce

1. Navigatea to https://app.unkey.com/apis
2. Press `Tab` key
3. Observe

### Expected behavior

I was expecting some border around image that it's active, focused but I don't.

### Version info

```markdown
- OS: Windows
- Browser: Chrome
```

**Comment by chronark** [comment]
yeah, we should focus on more keyboard support in the upcoming redesign
